### PR TITLE
Unorphans the blackbox

### LINKED ISSUE #26130: Testmerged PRs logging appears to be broken?
I might just be missing something though.

`SELECT details FROM ss13feedback WHERE var_name = 'testmerged_prs';` => 0 results

### PR DESCRIPTION
**Pull request by Cyberboss**
Wew, git was actually able to track the rename even though it's a 100% diff

Moved sql_poll_population from server_maint to this new SS. Moved few remaining server_maint tasks to SSping and deleted it.

@nfreader 

Fixes #26130 The blackbox was being instantiated after revdata

### CODE REVIEW

**Comment by nfreader** [comment]
Won't this conflict with #26116?

**Comment by Cyberboss** [comment]
Yeah, we'll deal with that when it happens

**Comment by XDTM** [comment]
Unrelated but since a blackbox was mentioned, could it be possible to log persistence items like in secret satchels?

**Comment by Jordie0608** [comment]
As with #26134, why? 
Moving the remaining tasks into SSping means the name is wrong as it's not just handling the ping checks but also other server tasks; maintenance things, for the server. Do you get my drift here =P

**Comment by Cyberboss** [comment]
Moving it here fixes a bug with feedback being set before the blackbox inits, since subsystems come first. See #26132. Maybe I should move ping into server tasks?

**Comment by Jordie0608** [comment]
If they're going to be combined I think it'd make more sense like that, so long as they both plays nice with different flags and wait times.

**Comment by Jordie0608** [comment]
Conflict

File: code/controllers/subsystem/server_maint.dm [hunk 1]
```diff
@@ -1,23 +1,39 @@
...
 SUBSYSTEM_DEF(server_maint)
 	name = "Server Tasks"
-	wait = 6000
-	flags = SS_NO_TICK_CHECK
+	wait = 6
+	flags = SS_POST_FIRE_TIMING|SS_FIRE_IN_LOBBY
+	priority = 10
+	var/list/currentrun
 
-/datum/controller/subsystem/server_maint/Initialize(timeofday)
```
Comment (optimumtact) [inline code/controllers/subsystem/server_maint.dm]: was that value used for anything in this call or the parent calls?

Comment (Cyberboss) [inline code/controllers/subsystem/server_maint.dm]: The parent call gets it either way because of `..()`

Comment (optimumtact) [inline code/controllers/subsystem/server_maint.dm]: you removed it

Comment (Cyberboss) [inline code/controllers/subsystem/server_maint.dm]: ..() calls the parent with the original args

Comment (RemieRichards) [inline code/controllers/subsystem/server_maint.dm]: All arguments available to a proc should be present in all overrides, even if not used, THAT is enough of a reason to revert this change.

**Comment by Cyberboss** [comment]
>Stale
>Been ready a full week

REEEEEEEEEE
